asdfghjkl/precondition.py

```diff
@@ -86,7 +86,10 @@ def accumulate_curvature(self, postfix='acc', scale=1., to_pre_inv=False, replac
             if fisher_acc is None or replace:
                 self._set_fisher(module, fisher, postfix)
             else:
-                self._set_fisher(module, fisher_acc + fisher, postfix)
+                if ema_decay:
```

Comment:
if ema_decay < 1:

might be better. we want to skip this part if `ema_decay==1` (the default value of `self.ema_decay`)